### ISSUE TITLE
Seizure cluster

### ISSUE THREAD

**Issue by oliviajhenry** [obophenotype/human-phenotype-ontology#6110]
**Preferred term label:**
Seizure cluster

**Synonyms**
Acute repetitive seizures, seizure flurries, cyclical seizures, serial seizures, crescendo seizures, recurrent seizures.

**Definition (free text, please give PubMed ID)**
A fourfold or greater increase of usual seizure frequency within a 3-day period, where the patient demonstrates full recovery between seizure events within the cluster. The seizure cluster should be recognisable by the primary clinician, carer or patient.

**Comment**
There is no consensus within the medical community regarding the definition of a seizure cluster (see PMID: 29871784), where limitations of current clinical definitions include poor sensitivity and specificity. The proposed definition is meant to serve as a placeholder until a more robust definition, likely facilitated by technological advances in seizure tracking devices and personalised cluster identification (PMID: 32305858), is agreed upon.

**Parent term (use hpo.jax.org/app)**
Seizure (HP:0001250)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #300088

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@DLewisSmith please advise

**Comment by DLewisSmith** [comment]
Good idea @oliviajhenry.

@pnrobinson, This is a recognisable temporal feature of seizures that seems to have distinguishing value (most classically in _PCDH-19_-related epilepsies, but also others). Despite inconsistent definitions, most epileptologists know what we mean by it, and with clarification of the time-frame, so do patients/families. I agree that its inclusion within the HPO would be useful.

I like @oliviajhenry's definition, but might add 'The seizures may be of any type' explicitly. I am unsure whether it is best placed under "Seizure HP:0001250" or "Recurrent HP:0031796" as it is not a type of seizure, rather a temporal pattern. I do not think that "Periodic HP:0025304" or "Episodic HP:0025303" would be sufficient as the individual seizures could occur regularly or irregularly within a cluster and the clusters could occur at either regular or irregular intervals. Hence, a new term (presumably as a child of "Episodic", capturing its extreme state) and definition would be required for "Clustering", such as "A transient sign, symptom, or other manifestation that recurs irregularly but on occasion in a highly concentrated manner." Conversely, placing it under "Seizure HP:0001250" would be simple in that the definition is can remain seizure domain-specific rather requiring modification to be acceptable more generically (to the point that the exact temporal parameters become unclear). @pnrobinson what do you think?

I have reservations about some of the suggested synonyms. 'Acute repetitive' might be interpreted to mean a single provoked episode of multiple seizures at/soon after a head injury/stroke/hypoglycaemia etc., 'cyclical' might be used to describe periodic (regularly spaced) single (rather than necessarily repeated) seizures and would be consistent with "Periodic HP:0025304", for example in catamenial epilepsy, 'Recurrent' and 'Serial' do not necessarily suggest an short-term increase in seizure frequency. I do like 'Cluster' in the preferred name, and 'Flurries', and 'Crescendo' in the synonyms. Is this reduced list of synonyms acceptable to you @oliviajhenry?

@pnrobinson, side notes: I note that 'Cyclic(al)' search does not currently bring up 'Periodic', I wonder if it should. I also note that "Generalized tonic-clonic seizures on awakening HP:0007193" has been retained, which seems reasonable if "Seizure cluster" is introduced in the "Seizure HP:0001250" subontology. However, it would need a new definition – I'll put a separate requests in for these.

**Comment by pnrobinson** [comment]
@DLewisSmith  I am wondering if it would be better to create modifiers to describe the frequencies etc of seizures -- these could then be applied to any type of seizure

seizure modifier
--seizure duration
----1 minute seizure duration
----whatever terms are needed
--seizure triggers
----whatever needed
--seizure time of day at onset
-----morning
-----sleep
--seizure frequency
----once a day
----whatever
--seizure distribution
----random distribution
------seizure cluster

or something like that. This would avoid us having to create hundreds and hundreds of new terms to describe things like seizure clusters for each seizure type.
The disadvantage is that it makes things more complicated -- you need two terms
The advantage is that it keeps the ontology simpler and it is easier to mix and match

**Comment by DLewisSmith** [comment]
Interesting thought for a future project. I'll give it some thought, @oliviajhenry, if this interests you maybe we could look at it together at some point. Diurnal variation, frequency, and distribution are of particular interest. Triggers are partially covered in the reflex seizure subontology. A change would be the variation in these, both natural/intrinsic and that resulting from interventions. We often have to spend quite some time in clinic clarifying, or personalising questions to get a good grasp of a patient's sz frequency.

Clustering is a relatively discrete one, so I think a term for this would be reasonable as a start.

**Comment by oliviajhenry** [comment]
Hi @DLewisSmith and @pnrobinson. I think creating a new node to describe seizure modifiers would be very useful. I'd be happy to look further into this together sometime.  @DLewisSmith I agree that some of the initial synonyms could lead to incorrect assignment of the term, and like your addition to the definition. Here's the updated synonym list and definition which currently stands:

**Synonyms**
Seizure flurries, crescendo seizures

**Definition (free text, please give PubMed ID)**
A fourfold or greater increase of usual seizure frequency within a 3-day period, where the patient demonstrates full recovery between seizure events within the cluster. The seizure cluster should be recognisable by the primary clinician, carer or patient. The seizures may be of any type.

**Comment by pnrobinson** [comment]
@DLewisSmith -- so just to clarify, I will add the following under CLinical modifier
https://hpo.jax.org/app/browse/term/HP:0012823
Seizure characteristic
--Seizure cluster (with definition as above)
(later followed by other terms?)

**Comment by DLewisSmith** [comment]
If you think this better than under "Seizure" or "Episodic" then yes. We have already have concepts such as triggers (for reflex seizures) and duration (Status epilepticus and complex febrile seizures) in the Seizure subontology so we'd need to think carefully about how these link up if we pull them under "Seizure characteristic" in the future.

I also wonder if "modifier" may be better than "characteristic", as some would say that all the semiologies in the "Seizure" subontology are also characteristics of the seizure.

**Comment by pnrobinson** [comment]
added to the 'Recurrent' clinical course subhierarchy as discussed.

**Comment by PabloBotas** [comment]
Hi,
I just noticed that *seizure cluster* is now under *recurrent*, which is under *temporal pattern* in *clinical course*. Why is the term not simply "cluster" and then it can be applied to any symptom of the ontology? Are there no other symptom that can be presented in temporal clusters? Maybe simplistic and too close to a technical view rather than a medical practitioner's view. It's surprising to see *seizure cluster* under this subontology.

Also, modern NLP algorithms can detect modifiers and it would be feasible to detect the relation of the term cluster in relation with, for instance, nocturnal seizures.

Thanks!

**Comment by pnrobinson** [comment]
Hi Pablo, our general rule has been to precompose (i.e., in this case add the "cluster" modifier) if there is a more specdific meaning that just that which we would get from the general terms. In this case, we have 

A fourfold or greater increase of usual seizure frequency within a 3-day period, where the patient demonstrates full recovery between seizure events within the cluster. The seizure cluster should be recognisable by the primary clinician, carer or patient. The seizures may be of any type.

Thus, there is a specific meaning for "cluster" in this context that we would not have if we said
(Hpo Term for Seizure) plus (Hpo Modifier term for Recurrent)

Practically speaking, this is also a term that we want users to annotate if applicable, and users might not think of using a modifier term

**Comment by pnrobinson** [comment]
@DLewisSmith would it make sense to have a "cluster occurence" modifier in addition to or instead of this term?

**Comment by DLewisSmith** [comment]
At some point probably, but I wouldn’t prioritise it yet.

**Comment by PabloBotas** [comment]
Thank you both for your explanation. I understand the logic here. The only issue I had is that it is surprising to see *seizure cluster* as such as special case of a general term such as recurrence.

Also, this design might pose problems for automatic phenotype extractors. NLP algorithms would identify *seizure cluster* as a modifier, not as a symptom. So sentences like *The patient presented seizure clusters* might be tagged only with a modifier. This would depend on the algorithm in question, but it does pose some issues.

Anyway, I just wanted to bring this up. I think the *cluster occurence* suggestion instead of the current term is perfect from the ontology perspective.

Thanks!

**Comment by DLewisSmith** [comment]
Yes, I agree with all the above. My concern about _cluster occurrence_ would be how to define it explicitly, as the definition of a cluster may depend on the context (and thus, the term it is intended to modify). I can foresee situations where the EHR includes something such as "She had a cluster of seizures earlier this year...", which we would think of as having a different (more generic) meaning of _cluster_ than "His nocturnal seizures occur in clusters". I think this is an example of how terminology in epileptology can be rather ambiguous without context and domain knowledge and poses interesting challenges to NLP. Beyond this, I imagine many paroxysmal events can cluster and the meaning/definition would depend on the clinical domain context (within neurology alone this would include timeframes spanning seconds to years).

At some point we should think more about the seizure modifiers you suggested above @pnrobinson, as I think this would be an interesting avenue to explore for standardising our seizure data. There are efforts to develop common data elements in the EHR that we could map. However, I will probably be too busy until end of 2022.

**Comment by PabloBotas** [comment]
Wouldn't seizure cluster be equivalent to sequential seizure here? It looks to me that it is more than just a modifier. It looks like it is its own entity. Source: [Classification-of-seizures---modification-for-neonates-epi.16815-2021-02.pdf](https://github.com/obophenotype/human-phenotype-ontology/files/7880584/Classification-of-seizures---modification-for-neonates-epi.16815-2021-02.pdf)

If this is the case then it should be just another seizure type.

**Comment by DLewisSmith** [comment]
It's the full recovery that would distinguish them, even among neonates.

By my understanding, a "neonatal sequential seizure" is a continuing prolonged seizure and evolving through different seizure types in which it is difficult to know where one ends, any brief period when the infant is not hoeing seizure, and when the next begins. It may be that some of these would be considered status epilepticus, but the use of this term in neonates is unclear to me (I'd need to speak to collaborators involved in neonatal classification - the neonatal brain has a different seizure repertoire to older children/adults). Hence, we have not linked this to status epilepticus for the same reasons we have not linked "Neonatal electro-clinical myoclonic seizure" to myoclonic seizure _etc._ However, this suggestion may come, I need to discuss it with others who have greater knowledge of neonatal seizures.

A "Seizure cluster" is differentiated from different to both status epilepticus and sequential seizures. The individual has recovery between seizures, often for hours or days (depending on context and definition used) where as this is not the case in "Status epilepticus" (which can either be one continuing prolonged seizure, or several seizures recurring without recovery between). "Neonatal sequential seizures" have no discernible recovery between different components. Additionally, if one goes by the ILAE classifications and definitions, these only apply to neonates. The alternative "4-Dimensional classification" considers seizures as concatenated components which could be described as sequences but this cannot be completely harmonised within the HPO structure (we have tried to integrate what concepts we can).

A question that might be relevant is whether Seizure cluster should _also_ be a child of seizure. This would allow the presence of "Seizure" to be inferred from it in the absence of any seizure sub-ontology term. However, I this would risk superficial coding of seizure phenotypes if a coder simply codes the seizures as a "seizure cluster" rather than giving more _informative_ seizure types.

**Comment by pnrobinson** [comment]
@DLewisSmith we are going to be making a new release next week and I am trying to add outstanding terms. My impression is that this one might need more thought first? At some point we could consider working on seizure modifying terms, but this is not a simple topic.

**Comment by pnrobinson** [comment]
@DLewisSmith  would it make sense to add some terms to this modifier hierarchy to capture this?
https://hpo.jax.org/app/browse/term/HP:0011008

**Comment by pnrobinson** [comment]
added as modifer term, HP_0033349
